ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -70,12 +102,42 @@ const Variant = ({ variant }) => {
 
 Variant.propTypes = {
   variant: PropTypes.object,
+  isCompoundHet: PropTypes.bool,
 }
 
 
+const CompoundHets = ({ variants }) => {
+  const allGeneIds = getCompoundHetsAllGeneIds(variants)
+  const sharedGeneId = getCompoundHetsSharedGeneIds(allGeneIds)
+  return (
+    <StyledVariantRow key={variants[0].variantId} >
+      {variants[0].familyGuids.map(familyGuid =>
+        <Grid.Column key={familyGuid} width={16}>
+          <FamilyVariantTags familyGuid={familyGuid} variant={variants} key={variants[0].variantId} areCompoundHets />
+        </Grid.Column>,
+      )}
+      <Grid.Column width={16}>
+        {sharedGeneId &&
+        <VariantGene geneId={sharedGeneId} variant={variants[0]} areCompoundHets />}
+      </Grid.Column>
+      <StyledCompoundHetRows stackable columns="equal">
+        {variants.map(variant =>
+          <Variant variant={variant} key={variant.variantId} isCompoundHet />,
+        )}
+      </StyledCompoundHetRows>
+    </StyledVariantRow>
+  )
+}
+
+
+CompoundHets.propTypes = {
+  variants: PropTypes.array,
+}
+
 const Variants = ({ variants }) =>
   <Grid stackable divided="vertically" columns="equal">
-    {variants.map(variant => <Variant key={variant.variantId} variant={variant} />)}
+    {variants.map(variant =>
+      (variant.length > 1 ? <CompoundHets variants={variant} key={variant[0].variantId} /> : <Variant variant={variant} key={variant.variantId} />))}
```

Comment:
for the compound het key, you need to actually include both variant IDs because you theoretically could end up with multiple pairs of compound hets with the same first variant